src/redux/userReducer.js

```diff
@@ -0,0 +1,46 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+
+const TOKEN =
+  "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjYxMjNmMTdlNWMzY2IxNDVjMDQ1ZTk0ZSIsInJvbGUiOiJhZG1pbiIsImlhdCI6MTYzMDk1NTk0NCwiZXhwIjoxNjMxODM0MzQ0fQ.AXc0KhpkkR9d475zpfQVH1nXugZ1SJomyQ1uJBSerRw";
+
+export const userFetch = createAsyncThunk(
+  "user/fetch",
+  async function (_, { rejectWithValue }) {
```

Comment:
it's better to use separated 3 actions for request handling, it looks more readable and workable than `rejectWithValue`...

```
FETCH_CURRENT_USER_REQUESTED - to send a request
FETCH_CURRENT_USER_SUCCEEDED - to show that request is done successfully 
FETCH_CURRENT_USER_ERROR - failed request
```